ports/directxmath/vcpkg.json

```
@@ -0,0 +1,9 @@
+{
+  "name": "directxmath",
+  "version-string": "jan2021",
+  "port-version": 1,
+  "description": "DirectXMath SIMD C++ math library",
+  "homepage": "http://go.microsoft.com/fwlink/?LinkID=615560",
```

```suggestion
  "homepage": "https://github.com/Microsoft/DirectXMath",
```
The URL will be redirected to https://github.com/Microsoft/DirectXMath, we can use it directly here.

Note: Seems you just update CONTROL file as manifest for `directxmath`, there is no update for source codes.

---

If you'd rather use direct than fwlinks, that's fine by me...

---

DirectXMath was just a port update to vcpkg.json so I could include the extra metadata...